crates/pop-cli/src/commands/call/parachain.rs

```diff
@@ -362,6 +380,30 @@ fn get_param_value(
 	}
 }
 
+// Prompt for the value when is a primitive.
+fn prompt_for_sequence_param(cli: &mut impl Cli, param: &Param) -> Result<String> {
+	if cli
+		.confirm(format!(
+			"The value for `{}` might be too large to enter. Provide a file instead?",
```

Comment:
Let's maybe ask directly for a file instead of asking if the user wants to paste in the terminal. I think both of these, the code and the state, will be always better managed in files.

Reply:
My approach was similar to PolkadotJS, using a simple input example like the `remark` extrinsic where you probably won't need a File. However, your suggestion likely makes more sense, as it helps avoid potential input issues for users.

Reply:
Based on both opinions and to minimize unnecessary prompts, I removed the confirmation step. Now, the user input can be either a file path or a value, and `pop-cli` will handle the value:
```rs
cli
        .input(format!(
            "The value for `{}` might be too large to enter. You may enter the path to a file instead.",
            param_name
        ))
        .placeholder(&format!(
            "Enter a value of type {} or provide a file path (e.g., /path/to/your/file.json)",
            param_type
        ))
        .interact()?;
```
Refactored in this PR: https://github.com/r0gue-io/pop-cli/pull/363/commits/f39900e7f0791e61b0ad289333e01c54abe48bf2